playbooks/deploy.yaml

```
@@ -3,35 +3,38 @@
   hosts:
     - quadlet
   become: true
+  vars_files:
+    - "../vars/{{ certificate_source }}_certificates.yml"
```

```suggestion
    - "../vars/{{ certificate_source | default('default') }}_certificates.yml"
```
Then you don't have to pass the default (default).

---

Better, probably:
```suggestion
    - "../vars/{{ certificate_source }}_certificates.yml"
  vars:
    certificate_source: default
```

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable says `-e certificate_source=installer` would still win, if set